client/my-sites/stats/stats-purchase/stats-purchase-personal.tsx

```diff
@@ -11,22 +13,27 @@ interface PersonalPurchaseProps {
 }
 
 const AVERAGE_PRICE_INFO = '$6';
+const MAX_SLIDER_PRICE = 10;
+const EMOJI_HEART_TIER = 5;
```

Comment:
Could we note that we'll want to pull the pricing slider configuration values from the API in a future PR?

p1688673993777489/1688595086.573129-slack-C0438NHCLSY